app-thunderbird/src/main/kotlin/net/thunderbird/android/featureflag/TbFeatureFlagFactory.kt

```diff
@@ -7,7 +7,7 @@ import app.k9mail.core.featureflag.FeatureFlagKey
 class TbFeatureFlagFactory : FeatureFlagFactory {
     override fun createFeatureCatalog(): List<FeatureFlag> {
         return listOf(
-            FeatureFlag(FeatureFlagKey("material3_navigation_drawer"), false),
+            FeatureFlag(FeatureFlagKey("material3_navigation_drawer"), true),
```

Comment:
This looks like an unintentional change. It will lead to the app crashing because the new drawer implementation isn't available yet.

Reply:
Good catch